src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/TextsController.cs

```diff
@@ -0,0 +1,53 @@
+using Altinn.App.Services.Interface;
+using Altinn.Platform.Storage.Interface.Models;
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace Altinn.App.Api.Controllers
+{
+    [Route("{org}/{app}/api/texts/{language}")]
```

Comment:
Is there a reason this is not versioned?

Reply:
added versioning